qiita_db/user.py

```diff
@@ -266,9 +263,9 @@ def verify_code(cls, email, code, code_type):
         elif code_type == 'reset':
             column = 'pass_reset_code'
         else:
-            raise IncompetentQiitaDeveloperError("code_type must be 'create'"
-                                                 " or 'reset' Uknown type "
-                                                 "%s" % code_type)
+            raise IncompetentQiitaDeveloperError(
+                "code_type must be 'create' or 'reset' Uknown type %s" %
```

Comment:
This is not for this PR, and we probably want to open an issue, but, if there are only 2 options `create` and  `reset`, can we represent this as a bool? If `True` means reset, if false means create...